pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -148,7 +148,9 @@ func networksToHotplugWhoseInterfacesAreNotInTheDomain(vmi *v1.VirtualMachineIns
 		func(ifaceStatus v1.VirtualMachineInstanceNetworkInterface) bool {
 			_, exists := indexedDomainIfaces[ifaceStatus.Name]
 			vmiSpecIface := netvmispec.LookupInterfaceByName(vmi.Spec.Domain.Devices.Interfaces, ifaceStatus.Name)
-
+			if vmiSpecIface == nil {
```

Comment:
I do not understand how this can ever happen.

In what scenario this happened?

Reply:
Thanks for the review!
I was testing hotplug/unplug on v1.1.0 with c25e0f03149e1f9eae93337aed4513e53a85013c backported, not sure if this occurs on main.
This could be reproduced by:
1. hotplug a Multus interface (patch vm)
2. delete vmi and wait for vmi to be recreated
3. hot unplug the interface (set absent)
Marked as draft for now, will get back once I have the environment to reproduce.